src/containers/rides/createRidesContainer.js

```diff
@@ -0,0 +1,92 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import Ride from '../../components/rides/createRides';
+import Navigation from '../navigation/navContainer';
+import createRidesAction from '../../actions/rides/createRidesAction';
+
+class RideOffer extends React.Component {
+	constructor(props) {
+		super(props);
+		this.state = {
+			meetingpoint: '',
+			meetingpointError: '',
+			departure: '',
+			departureError: '',
+			destination: '',
+			destinationError: '',
+			slots: '',
+			slotsError: ''
+		};
+		this.handleChange = this.handleChange.bind(this);
+		this.handleSubmit = this.handleSubmit.bind(this);
+	}
+
+	validateUserData = () => {
+		this.setState({
```

Comment:
do we need to start by setting state here? @Estaer

Reply:
... Reason being from initial state we already have the error values as empty strings